types/node/buffer.d.ts

```diff
@@ -205,7 +205,7 @@ declare module "buffer" {
          * Returns a new `ReadableStream` that allows the content of the `Blob` to be read.
          * @since v16.7.0
          */
-        stream(): WebReadableStream;
+        stream(): WebReadableStream<Uint8Array>;
```

Comment:
Narrowing the stream element type from `any` to `Uint8Array` is a breaking change, not just because of the element type itself, but also because the read result value becomes potentially undefined, in a similar vein to the discussion around built-in iterator return types for TS 5.6:

```ts
declare const stream: ReadableStream;
declare const typedStream: ReadableStream<Uint8Array>();

stream.getReader().read().then(result => Buffer.from(result.value)); // OK
typedStream.getReader().read().then(result => Buffer.from(result.value)); // TS2769: No overload matches this call
```

I would suggest that this is left as-is for now, but added to the growing todo list for the next major @types/node release.
```suggestion
        // TODO: narrow stream parameter type to Uint8Array in next major version
        // stream(): WebReadableStream<Uint8Array>;
        stream(): WebReadableStream;
```